README.md

```diff
@@ -117,25 +121,24 @@ func _on_SyndiBox_signal_tag(identifier):
 `Path B`  
 
 ## Custom Tags
-SyndiBox allows you to create your own custom tags without overwriting any of the main addon code. The token for custom tags is `X`. For example `[X!]` or `[Xm]` or even `[X ]` are all valid custom tags. In order to do create a custom tag you should do the following:  
+SyndiBox allows you to create your own custom tags without overwriting any of the main addon code. The token for custom tags can be anything you want. For example `[X!]` or `[Magenta]` or even `[ 2 1 ]` are all valid custom tags. The only thing tags need is to start with '[' and end with ']', it also must not be empty ([if you wish to test it out click here](regexr.com/5ei0l)). In order to create a custom tag you should do the following:  
 - Find `custom.gd` in the addons folder  
 - Optionally create your own copy of `custom.gd` (or name it anything else you want) outside of the addon folder, and set the `Custom Effects` variable in the inspector to this new file. This allows you to avoid overwriting your custom work when you upgrade this plugin.
 - Add a new case to the match statement
-- It is **critical** that you include `string.erase(sb.step,4)` and `string = string.insert(sb.step,char(8203) + "[:2][^r]")` if you do not wish to break encoding and nested tagging
+- It is **critical** that you include `string.erase(sb.step,sb.emph.length())` and `string = string.insert(sb.step,char(8203))` if you do not wish to break encoding and nested tagging
 - `sb` is the SyndiBox parent, and from there you can access any variables you wish to change, including but not limited to, `color`, `speed`, `font`, `timer`, and many more.
 
 While the custom.gd has a couple good examples already, another simpler example is included below for reference:  
 ```gdscript
 func check(string):
  match sb.emph:
-  "[Xm]": # Example
-   if !sb.escape:
-    string.erase(sb.step,4)
-    string = string.insert(sb.step,char(8203) + "[:2][^r]")
+  "[Magenta]": # Example
+    string.erase(sb.step,sb.emph.length())
+    string = string.insert(sb.step,char(8203))
     sb.color = Color(255, 0, 255, 1)
 ```  
-This very simple example turns the text magenta when you use the tag `[Xm]` in a Dialog string.  
-As custom tags are checked and processed last, you `X` tags will override any other tags if they conflict. 
+This very simple example turns the text magenta when you use the tag `[Magenta]` in a Dialog string.  
+As custom tags are checked and processed first, your custom tags will be overriden any other tags if they conflict. 
```

Comment:
Will these custom tags _override_ or _be overriden_? It seems like a very big difference in wording and I'd like some clarification before I approve.

Reply:
I didn't write the code, but looking at the code, custom code is definitely overwritten as it processes first. I mistakenly wrote 'overrides' in the previous version of the readme based on the comments of the code which were out dated, not the code itself. Trust the guy who wrote it (him) not the guy who just tried to document it (me). Relevant code below.

```gdscript
func emph_check(string): # Called before printing each step
	# Checks if if the start of a tag
	if string[step] == "[":
		# Search for the entire tag
		var tag_match = tag_checker.search(string,step)
		# If it is found then saves it to emph
		if tag_match != null:
			emph = tag_match.get_string()
			# Attempt a match for a valid tag in our current string.
			if CUSTOM_EFFECTS:
				string = custom.check(string)
			string = speaker_check(string)
			string = font_check(string)
			string = color_check(string)
			string = speed_check(string)
			string = pos_check(string)
			string = pause_check(string)
			string = hide_check(string)
			string = signal_check(string)
```

Reply:
Yeah, I had custom tags set first so people could use it as a base for custom presets (i.e., characters or custom/multiple effects), and they are checked first, and overridden by any tags following it before the reset tag. So, if I were to write `[X:][\`c]`, any custom color I set would be overridden by the built-in tag for red.